common/source/docs/common-uavcan-setup-advanced.rst

```diff
@@ -100,6 +100,11 @@ UAVCAN Rangefinder configuration
 
 Set ``RNGFNDx_TYPE`` = 24 to enable UAVCAN rangefinder type. Rangefinder data received over UAVCAN will only be used if the received sensor_id matches the parameter ``RNGFNDx_ADDR``. For AP_Periph firmware based adaptor nodes, this value is 0, so ``RNGFNDx_ADDR`` must be set to 0. Other UAVCAN rangefinders may differ. See also :ref:`UAVCAN Adaptor Node<common-uavcan-adapter-node>` instructions.
 
+UAVCAN Node ID Conflicts
+========================
+
+When a device is attached and recognized, it's node ID and hardware ID are entered into a database which is stored between power cycles. If multiple devices with the same ID and different hardware IDs are used (swapping smart batteries, for example, without cycling power), a conflict will arise in the database. This will require the use of the :ref:`CAN_D1_UC_OPTION<CAN_D1_UC_OPTION>` parameter to allow the database to be reset on the next boot, or conflicts in the database to be ignored.
```

Comment:
change "swapping smart batteries, for example, without cycling power" to "swapping smart batteries with fixed node IDs, for example"

Reply:
done